includes/class-wc-stripe-customer.php

```diff
@@ -334,6 +334,38 @@ public function set_default_source( $source_id ) {
 		return false;
 	}
 
+	/**
+	 * Retrieves the default payment method either from the WC or Stripe customer.
+	 *
+	 * @since 4.3.1
+	 * @return object|null Either a basic source object, or null.
+	 */
+	public function get_default_source() {
+		if ( ! $this->get_id() ) {
+			return null;
+		}
+
+		// First, try to load the default source from the WooCommerce customer object.
+		if ( ! empty( $this->user_id ) ) {
+			$default_token = WC_Payment_Tokens::get_customer_default_token( $this->user_id );
+
+			if ( ! empty( $default_token ) && 'stripe' === $default_token->get_gateway_id() ) {
+				$source = WC_Stripe_API::request( array(), 'sources/' . $default_token->get_token(), 'GET' );
+				if ( empty( $source->error ) && $this->id === $source->customer ) {
```

Comment:
When checking `$this->id === $source->customer` I get this exception:

```
Exception has occurred.
Notice: Undefined property: stdClass::$customer
```

The `source` object I end up with does not have a `customer` property. It looks like this happens when the source is not attached to the customer:

https://stripe.com/docs/api/sources/object#source_object-customer

We need to make sure we check if `$source->customer` is set before using it

Also, I wonder if this is expected when adding credit cards using `My Account` for a Woo customer.